recOrder/cli/utils.py

```diff
@@ -8,15 +8,18 @@
 from iohub.ngff_meta import TransformationMeta
 from numpy.typing import DTypeLike
 
+CODEC_MAX_BYTES = 2147483647
```

Comment:
Magic numbers should either be derived from an expression or accompanied by a comment. For long numbers, the digits should be separated e.g. `2_147_483_647`.